### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by KavishShah09**

### CODE REVIEW

File: gateway/utils/syncman/syncman.go [hunk 1]
```diff
@@ -32,6 +35,11 @@ type Manager struct {
 
 	// For authentication
 	adminMan *admin.Manager
+
+	// Modules
```
Comment (YourTechBud) [inline gateway/utils/syncman/syncman.go]: Remove cb from the struct

File: gateway/utils/server/server.go [hunk 2]
```diff
@@ -185,47 +142,47 @@ func (s *Server) LoadConfig(config *config.Config) error {
 
 		// Always set the config of the crud module first
 		// Set the configuration for the crud module
-		if err := s.crud.SetConfig(p.ID, p.Modules.Crud); err != nil {
```
Comment (YourTechBud) [inline gateway/utils/server/server.go]: Remove the `LoadConfig` method. We don't need it anymore

File: gateway/utils/syncman/syncman.go (hunk 1 above)
Comment (KavishShah09) [reply gateway/utils/syncman/syncman.go]: Some functions still call s.cb so should I call SetProjectConfig instead at those places?

Comment (YourTechBud) [reply gateway/utils/syncman/syncman.go]: Which functions?

**Pull request merged**